### PR TITLE
revisions to using alt text article

### LINKED ISSUE #678: Article Audit: Using alt text properly
Taking on the task to review/update [Using alt text properly](https://a11yproject.com/posts/alt-text/), per #669.

### PR DESCRIPTION
**Pull request by scottaohara**
Fixes #678
* expand intro
* revise tips
* add reference links

### CODE REVIEW

File: _posts/2013-01-14-alt-text.md [hunk 1]
```diff
@@ -1,21 +1,29 @@
...
-- Images that are links should describe where they go. For example, a logo that is a link to your home page should have `alt="home page"`. But if the image is only one part of the contents of a link and the other contents (e.g. plain text) describes the link then use an empty `alt` attribute for the image - if you were to specify 'alt' text as well this would be redundant/repetitive.
+## A few tips on how and when to use the `alt` attribute:
+- Use the `alt` attribute for images that are used as more than decoration. The alternative text describes the primary purpose of an image and is rarely a detailed description of the image itself (unless, for example, the page is actually about a specific image).
+- Make sure the description of the image is useful. For example, if the image is of a company's logo, the `alt` should (often) be the company's name. The word "logo" is not necessary or useful as part of the alternative text.
```
Comment (ericwbailey) [inline _posts/2013-01-14-alt-text.md]: I think "(often)" here adds more complexity/confusion. What are the circumstances where you shouldn't when it comes to logos? How will I know if I should or should not?

File: _posts/2013-01-14-alt-text.md [hunk 2]
```diff
@@ -1,21 +1,29 @@
...
+- Don't begin alternative text with "photo of..." or "picture of..."; assistive technologies already indicate the role of the element as an "image" or "graphic". Unnecessary redundancy makes for poor user experiences.
```
Comment (ericwbailey) [inline _posts/2013-01-14-alt-text.md]: Replace the semicolon with a period.

Comment (ericwbailey) [inline _posts/2013-01-14-alt-text.md]: Remove "Unnecessary" as well (it's sorta redundant!).

File: _posts/2013-01-14-alt-text.md [hunk 3]
```diff
@@ -1,21 +1,29 @@
...
+- [Don't put line breaks in `alt` text](https://developer.paciellogroup.com/blog/2015/09/short-note-on-coding-alt-text/). It causes suboptimal effects when announced by screen readers.
```
Comment (ericwbailey) [inline _posts/2013-01-14-alt-text.md]: I'd rephrase this to use something other than suboptimal. It's kinda a jargon-adjacent term in an situation where the reader may not already know what alt text is supposed to sound like. Maybe describe what the effect would be, for those who don't read the link? That or just say something like, "It can break how a screen reader reads the image's description."

File: _posts/2013-01-14-alt-text.md [hunk 4]
```diff
@@ -1,21 +1,29 @@
...
+- Use an empty `alt` attribute for any image that is decorative or not necessary for understanding the content of the page (`alt=""` or just `alt` - e.g. `<img src="prettyswirl.jpg" alt>`). If, for example, the information in a graph is fully explained in text accompanying the image, there's no need to duplicate that in `alt` text.
```
Comment (ericwbailey) [inline _posts/2013-01-14-alt-text.md]: I'd take the examples of how to supply a null alt value out of the parenthetical statement. They're very much in-line with the main subject of this bullet point—might even make sense to directly state "For example, alt descriptions can be disabled in a screen-reader friendly way by using either…"

File: _posts/2013-01-14-alt-text.md [hunk 5]
```diff
@@ -1,21 +1,29 @@
...
+- Using an image several times in a website doesn't necessarily mean the `alt` attribute will be the same for each instance. For example, a logo within a website's header often doubles a link back to the home page. In this example, the `alt` would be useful as "Company's name - home". If that same logo were used in the footer of the site, next to the text "Company's name, copyright 20XX", then the logo would have an empty alt (`alt=""`) so as to not create a redundant announcement of the company's name.
+- There may be other times where images are used within links, but are decorative to the visible label of the link. For instance, if the image is an icon of a phone and part of a "contact" link, the `alt` could be empty.
```
Comment (ericwbailey) [inline _posts/2013-01-14-alt-text.md]: Use "For example," not "For instance,". You use instance in a more dev way in the above bullet point, which could muddy its meaning.

Comment (ericwbailey) [inline _posts/2013-01-14-alt-text.md]: I might specifically explain how this makes it decorative by telling the reader how the text label supplies the description.

File: _posts/2013-01-14-alt-text.md [hunk 6]
```diff
@@ -1,21 +1,29 @@
...
+
+## More resources for writing `alt` text
```
Comment (ericwbailey) [inline _posts/2013-01-14-alt-text.md]: Line break after subheadings!

File: _posts/2013-01-14-alt-text.md (hunk 1 above)
Comment (scottaohara) [reply _posts/2013-01-14-alt-text.md]: 'often' was used in reference to the logo being referenced differently in other examples.  but yeh, will remove for simplicity.
